### PR TITLE
Lowers log level to debug

### LINKED ISSUE #803: Suppress "ignoring compiler output js change..." message unless debugging enabled
### Describe the problem

If code in a component doesn't functionally change, but has updates in whitespace after saving, the plugin will emit a message such as this:
```
10:50:22 AM [vite-plugin-svelte] ignoring compiler output js change for /path/to/component.svelte as it
is equal to previous output after normalisation
```

This happens frequently when you reformat your code, perhaps even automatically in your IDE, so the console will contain a lot of these.  
I appreciate tooling letting me know what happens, but this message is neither actionable nor relevant. Is there any reason to emit it at all, unless debugging is enabled?

### Describe the proposed solution

I would like to suppress this log output, unless verbose or debugging mode is explicitly enabled.

### Alternatives considered

I know this issue is a nitpick, but improving DX is always good :)

### Importance

nice to have

### PR DESCRIPTION
**Pull request by Radiergummi**
This message isn't immediately actionable for developers and can reasonably be considered debugging output. Therefore, the log level has been updated to reflect this.

This fixes #803.

### CODE REVIEW

**Comment by Radiergummi** [comment]
@dominikg as requested; here goes. I was pretty confident you don't require tests for logging, so this PR is as bare-bones as it gets.

**Comment by dominikg** [comment]
the PR also needs a changeset, you can create it with `pnpm changeset` and following the prompts.  (select vite-plugin-svelte, select patch, write a short note like "changed loglevel of output unchanged message to debug to reduce verbosity")

**Comment by Radiergummi** [comment]
> the PR also needs a changeset, you can create it with `pnpm changeset` and following the prompts. (select vite-plugin-svelte, select patch, write a short note like "changed loglevel of output unchanged message to debug to reduce verbosity")

Changeset created, waiting for approval.

**Comment by Augasty** [comment]
Hi @Radiergummi and @dominikg, I was trying to solve the issue, found the bug and after solving it, I came to see that @Radiergummi has already raised a PR. 
I am not entirely sure whether I should raise my own PR (This is my first try at open source contribution, so please excuse me if I am doing something wrong). 

So, I am uploading my changes anyway, thanks.

**Comment by dominikg** [comment]
Thank you @Radiergummi !

@Augasty  raising a duplicate PR is usually not a good idea, but participating in issues and PRs is. Your PR itself looks fine too, but this one is going ahead as it was first.

**Comment by Augasty** [comment]
Absolutely okay with it. And thanks a lot for feedback.

**Review by dominikg — APPROVED** [review]

**Pull request merged**
